src/wfs/src/main/java/org/geoserver/wfs/WFSGetFeatureOutputFormat.java

```diff
@@ -229,6 +238,38 @@ protected int getNumDecimals(List featureCollections, GeoServer geoServer, Catal
         return numDecimals;
     }
 
+    protected boolean getPadWithZeros(
+            List featureCollections, GeoServer geoServer, Catalog catalog) {
+        boolean padWithZeros = false;
+        for (int i = 0; i < featureCollections.size(); i++) {
+            Boolean pad =
+                    getFeatureTypeInfoProperty(
+                            catalog,
+                            (FeatureCollection) featureCollections.get(i),
+                            fti -> fti.getPadWithZeros());
+            if (pad != null && pad.booleanValue()) {
+                padWithZeros = true;
+            }
+        }
+        return padWithZeros;
+    }
+
+    protected boolean getForcedDecimal(
+            List featureCollections, GeoServer geoServer, Catalog catalog) {
+        boolean forcedDecimal = false;
+        for (int i = 0; i < featureCollections.size(); i++) {
+            Boolean forced =
+                    getFeatureTypeInfoProperty(
+                            catalog,
+                            (FeatureCollection) featureCollections.get(i),
+                            fti -> fti.getForcedDecimal());
+            if (forced != null && forced.booleanValue()) {
```

Comment:
This is a bit weird way to do the comparison, ``Boolean.TRUE.equals(forced)`` would be the idiomatic approach